### DISCUSSION TITLE
Ideas de donde Hostear nuestras Web Apps Gratis (Sin Back)

### DISCUSSION THREAD

**Discussion by AlexisRamirezHernandez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#165] [General]
Hola viajerxs les comparto una info por si no saben por donde empezar y quieren publicar sus proyectos web en alguos host gratuitos. Con sus respectivas limitantes.
[Haz Click Aqui](https://www.instagram.com/p/CZx0o6sgHR_/)

**Comment by JorgeEHE** [comment]
Excelente aportación compañero, muchas gracias por el dato, lo tendré en cuenta cuando realice mi práctica y tenga que subir la página

**Comment by PaulHRios** [comment]
También pueden hacer host mediante GitHub para que no batallen. Aquí un tutorial de como hacerlo, es muy fácil.
https://www.youtube.com/watch?v=un5WjcP027Q&ab_channel=AlexCGDesign
